src/vsl/parser/grammar/ws.ne

```diff
@@ -1 +1,2 @@
-_ -> "\n":*
+_
+   -> "\n":*
```

Comment:
I wanted to maintain consistency so that's why it's on another line. Regarding your second question, my editor (vim) already handles that newline stuff and GitHub will say when there's no newline at the end via that little red symbol thing when you scroll to the bottom of it.